src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -359,6 +374,9 @@ def write_new_values_from_gui(self):
         if self.txtUpTheta.isModified():
             self.model.params['Up_theta'] = float(self.txtUpTheta.text())
 
+        if self.txtUpPhi.isModified():
+            self.model.params['Up_phi'] = float(self.txtUpPhi.text())    
```

Comment:
Can the text contain non-float values? If so, what happens when an exception is thrown? Verify that a reasonable message is shown to the user when bad values are entered.

Reply:
There are now validators for all the inputs to allow only reasonable values.